web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -745,7 +745,9 @@
           if (xhr.status === 200) {
             if (dataType === "script") {
               try {
+                /* jshint ignore:start */
                 eval(xhr.responseText);
+                /* jshint ignore:end */
```

Comment:
I think the only way is to turn off eval checking globally, which is probably fine. It's likely whoever will be editing this knows not to use eval unless it's necessary.